pkg/util/initsystem/initsystem.go

```diff
@@ -43,13 +43,13 @@ type SystemdInitSystem struct{}
 
 func (sysd SystemdInitSystem) ServiceStart(service string) error {
 	args := []string{"start", service}
-	_, err := exec.Command("systemctl", args...).Output()
+	err := exec.Command("systemctl", args...).Run()
```

Comment:
Not going to make you do this, since this was already this way, but it would probably be good to switch to `util.exec` for testability...

Reply:
maybe @bsteciuk can follow up later?